axelrod/tournament.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, players, match_generator=RoundRobinMatches,
         self.name = name
         self.turns = turns
         self.noise = noise
+        self.num_interactions = 0
```

Comment:
I thought about that but then it would only work for the case of generators for which we know the size. However doing it this way (counting the rows we write) ensures that the size of the data file is always known.